src/components/BlockQuote/BlockQuote.js

```diff
@@ -17,11 +17,12 @@ const styles = {
   })
 }
 
-const BlockQuote = ({ children, attributes }) => {
+const BlockQuote = ({ children, attributes, style }) => {
```

Comment:
Overwriting styles like this make maintenance hard. And when overwriting media query behaviour things get hard to understand.

IMO just write a wrapper with it's own margin in `components/CommentBody/web/Blockquote.js` everything else is not needed in the comment case anyhow.